reader.go

```diff
@@ -20,13 +20,23 @@ var errReaderClosed = errors.New("brotli: Reader is closed")
 // It is arbitrarily chosen to be equal to the constant used in io.Copy.
 const readBufSize = 32 * 1024
 
-// NewReader initializes new Reader instance.
+// NewReader creates a new Reader reading the given reader.
+// It is the caller's responsibility to call Close on the Reader when done.
```

Comment:
Reader doesn't even have a Close method, so this line doesn't make sense.

Reply:
Ops, didn't notice it.